doc/classes/ScrollContainer.xml

```diff
@@ -1,15 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ScrollContainer" inherits="Container" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A helper node for displaying scrollable elements such as lists.
+		A container used to provide a child control with scrollbars when needed.
 	</brief_description>
 	<description>
-		A ScrollContainer node meant to contain a [Control] child.
-		ScrollContainers will automatically create a scrollbar child ([HScrollBar], [VScrollBar], or both) when needed and will only draw the Control within the ScrollContainer area. Scrollbars will automatically be drawn at the right (for vertical) or bottom (for horizontal) and will enable dragging to move the viewable Control (and its children) within the ScrollContainer. Scrollbars will also automatically resize the grabber based on the [member Control.custom_minimum_size] of the Control relative to the ScrollContainer.
-		Works great with a [Panel] control. You can set [constant Control.SIZE_EXPAND] on the children's size flags, so they will upscale to the ScrollContainer's size if it's larger (scroll is invisible for the chosen dimension).
+		A container used to provide a child control with scrollbars when needed. Scrollbars will automatically be drawn at the right (for vertical) or bottom (for horizontal) and will enable dragging to move the viewable Control (and its children) within the ScrollContainer. Scrollbars will also automatically resize the grabber based on the [member Control.custom_minimum_size] of the Control relative to the ScrollContainer.
+		Works great with a [Panel] control. You can set [constant Control.SIZE_EXPAND] on the children's size flags, so they will upscale to the [ScrollContainer]'s size if it's larger (scroll is invisible for the chosen dimension).
```

Comment:
idk who added the [Panel] mention, but it does **not** work great with Panel. Instead you should use the container's own `panel` theme property.

Reply:
Okay, removed this line entirely.